ansible_runner/utils/capacity.py

```diff
@@ -22,16 +27,27 @@ def get_mem_in_bytes():
 
 
 def get_uuid():
-    try:
-        with open('/var/lib/dbus/machine-id') as f:
-            uuid = f.read()
-        return uuid.strip()
-    except FileNotFoundError:
-        try:
-            with open('/etc/machine-id') as f:
-                uuid = f.read()
-                return uuid.strip()
-        except FileNotFoundError:
-            error = ("Could not find /var/lib/dbus/machine-id or "
-                     "/etc/machine-id files, UUID undiscoverable.")
-            return error
+    uuid_directory = Path('/etc/ansible/facts.d')
```

Comment:
I'm curious why we picked this location. Is this a canonical place for this kind of file?

Reply:
Oh, I think I see why now. So that file will be read and sent back when the heartbeat runs? Neat.

Reply:
We needed a place. I picked a place that was ansibley. VERY open to a better place.

Reply:
Can someone point me to the place that we read this on the AWX side?